translations/pt-br/04-ferramentas-utilizadas.md

```diff
@@ -0,0 +1,142 @@
+
+### Começando com Git e GitHub
+
+Para começar com Git e GitHub, siga estes passos:
+
+1. **Instalar o Git**: Baixe e instale o Git no seu computador visitando o site oficial do Git (https://git-scm.com/) e seguindo as instruções de instalação para o seu sistema operacional.
+2. **Configurar o Git**: Após instalar o Git, abra um terminal ou prompt de comando e configure as informações do seu usuário Git executando os seguintes comandos:
+
+```bash
+git config --global user.name "Seu nome"
+git config --global user.email "seu.email@exemplo.com"
+```
+
+Substitua "Seu nome" pelo seu nome completo e "seu.email@exemplo.com" pelo endereço de e-mail que você usou para se inscrever no GitHub.
+
+#### Praticando Contribuições com o VS Code
+
+Para praticar o uso do GitHub, vamos passar por criar um repositório, clonar o repositório, fazer alterações e empurrar essas alterações.
+
+1. **Crie um repositório**: Faça login na sua conta do GitHub e navegue até [https://github.com/new](https://github.com/new) ou clique no ícone "+" no canto superior direito e selecione "Novo repositório" no menu suspenso.
+
+Escollha você mesmo como o proprietário do repositório, e digite um nome para o seu repositório. Para este exemplo, usaremos `practice-repository`
+
+![new repo screenshot](../../images/new-repo.png)
+
+Escollha se você quer que seja público ou privado. Você também pode escolher inicializar o repositório com um arquivo README, um arquivo .gitignore e/ou uma licença.
+
+Por exemplo, inicialize o repositório com um arquivo README. 
+
+2. **Clone o repositório**: Uma vez que você tenha criado um repositório, você pode cloná-lo para a sua máquina local executando o seguinte comando no seu terminal ou prompt de comando:
+
+```bash
+git clone https://github.com/username/repository-name.git
+```
+
+Substitua "username" pelo seu nome de utilizador do GitHub e "repository-name" por `practice-repository`.
+
+3. **Acesse o seu repositório no seu editor de código**: Para este exemplo, estamos usando o VS Code. Abra o VS Code, navegue até a guia de arquivos e abra o seu `practice-repository`.
+
+4. **Faça alterações e confirme**: Deve haver um arquivo no seu repositório: `README.md`. Dentro desse arquivo, abaixo de `# practice-repository`, digite 'hello world!' Após ter feito alterações, você pode prepará-las e confirmá-las.
+
+Abra o seu terminal e certifique-se de que está no diretório correto. Por exemplo, os meus projetos de codificação estão dentro da minha pasta `Projects`. Usando o meu terminal, eu acessaria este repositório executando este comando:
+
+```bash
+cd projects/practice-repository
+```
+
+Agora que você está no diretório correto, use os seguintes comandos para adicionar as alterações que você fez e escrever uma mensagem de confirmação com uma descrição das alterações:
```

Comment:
Again, same "commit" situation, now is "confirmar", thankfully it follows a previous translation :D